compiler/src/yul/mappers/expressions.rs

```diff
@@ -278,7 +279,12 @@ pub fn expr_bin_operation(context: &Context, exp: &Node<fe::Expr>) -> yul::Expre
                 }
                 _ => unreachable!(),
             },
-            _ => unimplemented!(),
+            BinOperator::FloorDiv => match typ {
```

Comment:
@cburgdorf do you think we need both `//` and `/`?

Reply:
I don't think we need `//`. My understanding is that Python provides this because it only provides one numeric base type and via `/` and `//` it lets you switch between *floating point division* and *integer division*. We currently only have *integer division* and if we ever add another base type to represent numbers with a decimal point then one would be free to use that datatype to perform such a division operations. So it is the chosen base type that would decide about it, not the operator.